src/main/java/ca/corefacility/bioinformatics/irida/ria/config/thymeleaf/webpacker/util/WebpackAssetsManifestDeserializer.java

```diff
@@ -0,0 +1,54 @@
+package ca.corefacility.bioinformatics.irida.ria.config.thymeleaf.webpacker.util;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+/**
+ * Deserializer for the webpack assets-manifest file
+ */
+public class WebpackAssetsManifestDeserializer extends StdDeserializer<WebpackAssetsManifest> {
+
+	public WebpackAssetsManifestDeserializer() {
+		this(WebpackAssetsManifest.class);
+	}
+
+	public WebpackAssetsManifestDeserializer(Class<?> vc) {
+		super(vc);
+	}
+
+	@Override
+	public WebpackAssetsManifest deserialize(JsonParser jsonParser, DeserializationContext deserializationContext)
+			throws IOException, JsonProcessingException {
```

Comment:
I have a warning here that `JsonProcessingException` is never thrown.  It's a subclass of `IOException` so the `JsonProcessingException` can be removed here.

Reply:
updated in 324a942